src/ApiService/ApiService/onefuzzlib/JobResultOperations.cs

```diff
@@ -0,0 +1,128 @@
+﻿using ApiService.OneFuzzLib.Orm;
+using Microsoft.Extensions.Logging;
+using Polly;
+namespace Microsoft.OneFuzz.Service;
+
+public interface IJobResultOperations : IOrm<JobResult> {
+
+    Async.Task<JobResult?> GetJobResult(Guid jobId);
+    JobResult UpdateResult(JobResult result, JobResultType type, Dictionary<string, double> resultValue);
+    Async.Task<bool> TryUpdate(Job job, JobResultType resultType, Dictionary<string, double> resultValue);
+    Async.Task<OneFuzzResult<bool>> CreateOrUpdate(Guid jobId, JobResultType resultType, Dictionary<string, double> resultValue);
+
+}
+public class JobResultOperations : Orm<JobResult>, IJobResultOperations {
+
+    public JobResultOperations(ILogger<JobResultOperations> log, IOnefuzzContext context)
+        : base(log, context) {
+    }
+
+    public async Async.Task<JobResult?> GetJobResult(Guid jobId) {
+        return await SearchByPartitionKeys(new[] { jobId.ToString() }).SingleOrDefaultAsync();
+    }
+
+    public JobResult UpdateResult(JobResult result, JobResultType type, Dictionary<string, double> resultValue) {
```

Comment:
It looks like `UpdateResult` is only used internally by `JobResultOperations` so we can make it private and remove it from the interface.

Reply:
Same goes for `TryUpdate`